systemtests/src/test/java/io/enmasse/systemtest/iot/isolated/registry/FileDeviceRegistryTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.iot.isolated.registry;
+
+import io.enmasse.iot.model.v1.IoTConfigBuilder;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static io.enmasse.systemtest.iot.DefaultDeviceRegistry.newFileBased;
+import static io.enmasse.systemtest.iot.DefaultDeviceRegistry.newInfinispanBased;
+import static io.enmasse.systemtest.utils.IoTUtils.assertCorrectRegistryType;
+
+class FileDeviceRegistryTest extends DeviceRegistryTest {
+
+    @Override
+    protected IoTConfigBuilder provideIoTConfig() throws Exception {
+        return new IoTConfigBuilder()
+                .withNewSpec()
+                .withNewServices()
+                .withDeviceRegistry(newFileBased())
+                .endServices()
+                .withNewAdapters()
+                .withNewMqtt()
+                .endMqtt()
+                .endAdapters()
+                .endSpec();
+    }
+
+    @Test
+    void testCorrectTypeDeployed () {
+        assertCorrectRegistryType("file");
+    }
+
+    @Test
+    void testRegisterDevice() throws Exception {
```

Comment:
Where is the benefit in patching this through from "test" to "do" methods?

Reply:
I would say it is in the fact of being aligned with other systemtest classes following this pattern, also this gives some flexibility to the children class in case some parameter passing is needed

Reply:
We are using this construction among all the tests in systemtest testsuite. So i wanted to match it :)